lms/djangoapps/support/static/support/jsx/entitlements/components/EntitlementForm/index.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, InputSelect, InputText, TextArea } from '@edx/paragon';
+
+
+class EntitlementForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const isReissue = props.entitlement !== null && props.entitlement !== undefined;
```

Comment:
You could also consider 

```javascript
const isReissue = !!prop.entitlement;
```

Reply:
Also, based on this comment

![image](https://user-images.githubusercontent.com/8136030/37843520-886c5dd6-2e9b-11e8-8249-d6c32b7438d3.png)

is `isReissue` the best name for this property or is `isActiveEntitlement` (or something else) better? I don't have much domain knowledge, just double-checking.

Reply:
I think the answer to this will be related to possible changes to the flow around how I pass/what I pass in as props around entitlement, and whether I need to explicitly pass a 'formType' prop.